issuing_using_did.md

```diff
@@ -200,3 +200,201 @@ This allow a user to "reserve" an address for the revocation store in the future
 
 - I noticed that we may move the "DNS-TXT" out of the data object for existing OA document as well without affecting the security of it
 - How do we allow a document to use both DID and DNS proof?
+
+### Notes 11/08/20
+
+Directly signed with DID with DNS binding
+
+```json
+{
+  "schema": "tradetrust/v1.0",
+  "data": {
+    "issuers": [
+      {
+        "name": "TradeTrust Demo",
+        "identityProof": [
+          {
+            "type": "DID",
+            "id": "did:ethr:0xE6Fe788d8ca214A080b0f6aC7F48480b2AEfa9a6",
+            "purpose": "DOCUMENT_ISSUANCE",
+            "revocation": {
+              "type": "REVOCATION_STORE",
+              "address": "0xabcd...1234"
+            },
+            "key": {
+              "id": "did:ethr:0xE6Fe788d8ca214A080b0f6aC7F48480b2AEfa9a6#owner",
+              "type": "Secp256k1VerificationKey2018",
+              "ethereumAddress": "0xe6fe788d8ca214a080b0f6ac7f48480b2aefa9a6",
+              "owner": "did:ethr:0xE6Fe788d8ca214A080b0f6aC7F48480b2AEfa9a6"
+            }
+          },
+          {
+            "type": "DNS-DID",
+            "location": "example.com"
+          }
+        ]
+      }
+    ]
+  },
+  "proof": {
+    "type": "DidGenericSignature",
+    "proofPurpose": "assertionMethod",
+    "signature": "<signed merkle root>"
+  }
+}
+```
+
+Issued via Document Store, identified via DNS through DID
+
+```json
+{
+  "schema": "tradetrust/v1.0",
+  "data": {
+    "issuers": [
+      {
+        "name": "TradeTrust Demo",
+        "documentStore": "0x6d71da10Ae0e5B73d0565E2De46741231Eb247C7",
+        "identityProof": [
+          {
```

Comment:
is this necessary in that case ? Can't we directly assume did = owner of doc store ? (to make it simpler for user)

Reply:
Also if we want to be backward compatible we will have to support it that way (automatically resolve the owner of the document store)

Reply:
For backward compatibility, you only have document store, you wont attempt to resolve that to any DID because there is none declared. It will just resolve directly to DNS

Reply:
Can't we directly assume did = owner of doc store ? (to make it simpler for user)
^ No you cant, that will break abstractions. Imagine if the `owner` of the document store is a proxy instead...

Reply:
ok then what's the point of showing an example with document store + did + dns ? (in term of use case, maybe i's just to show it's possible ?)

Reply:
yes, the same as an entity can have multiple identifiers